entries/k/kajabi.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Kajabi": {
+    "domain": "kajabi.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.kajabi.com/hc/en-us/articles/8293506425115",
+    "categories": [
+      "other"
```

Comment:
Could it be classified as `retail` and `marketing`?

Reply:
A service for retailers isn't really a retail service in and of itself.
I don't see that they offer marketing or analytics services either.